codespell_lib/data/dictionary.txt

```diff
@@ -6307,6 +6310,7 @@ coponent->component
 copoying->copying
 coppermines->coppermine
 coppied->copied
+copping->coping, copying, cropping,
```

Comment:
`copping` is a form of the verb `cop`, e.g., "copping an attitude" (not entirely obscure in English)

Reply:
good point. Though, this is why I for a 2ndary fallback dictionary accessible with a flag that can catch some of these more problematic typos (that are legit words)

Reply:
For now flag as `disabled ...` then? Then once we have the secondary dictionary support we can move it

Reply:
@larsoner 5959efe  
rebased as well.